### ISSUE TITLE
Default tile size

### ISSUE THREAD

**Issue by Nohus** [peterLaurence/MapCompose#28]
The default tile size is 256. Was there any research done in choosing it? What are the tradeoffs in making it smaller or bigger?

**Comment by peterLaurence** [comment]
A long time ago I played around with 512px and even 1024px tiles. I was hoping to have less computations and better performance. But that was too long ago to be meaningful today, and my app was in python (using the Kivy framework).
Today, all wmts tiles servers provide 256px tiles. I believe this is for historical reasons. It would be interesting to benchmark the library using various tile sizes. On one hand, with bigger tiles, we have less internal computations. But on the other hand we might waste more memory loading areas which aren't visible.

**Comment by Nohus** [comment]
I will try to benchmark different tile sizes when I have some time and post any results here.

**Comment by thegrxp** [comment]
Have you tested different tile sizes?

**Comment by Nohus** [comment]
Yeah, and I found the 256x256 tiles working best for me. Bigger tiles felt less responsive since each tile took more time to load (even though the full map took about same time overall). Smaller tiles just resulted in an excessive amount of requests for no real gain. It all depends on your use case though.
